sdk/storage/azure-storage-file-datalake/azure/storage/filedatalake/_shared/constants.py

```diff
@@ -5,10 +5,11 @@
 # --------------------------------------------------------------------------
 
 import sys
-from .._generated.version import VERSION
 
+from .._generated import AzureDataLakeStorageRESTAPI
 
-X_MS_VERSION = VERSION
+
+X_MS_VERSION = AzureDataLakeStorageRESTAPI(url="get_api_version")._config.version  # pylint: disable=protected-access
```

Comment:
can we ask isabella if we can add VERSION to generated code?

Reply:
Yes we can have this discussion offline with them (non-blocking)